wallet/src/App.vue

```diff
@@ -109,6 +125,10 @@ export default defineComponent({
     };
   },
   methods: {
+    async setCurrentAddress(e: any) {
```

Comment:
What if undefined value came here? We should have safe check here.
`If(e.target.value){// do something}`

Reply:
Done